python/tests/test_ring.py

```diff
@@ -70,6 +73,28 @@ def test_harminv(self):
         self.assertAlmostEqual(ep, 11.559999999999999, places=places)
         self.assertAlmostEqual(fp, -0.08185972142450348, places=places)
 
+    def test_pade(self):
+        self.init()
+
+        self.sim.run(
+            mp.at_beginning(mp.output_epsilon),
```

Comment:
Why is it necessary to output epsilon as part of the test?

Reply:
Probably not necessary. Out of curiosity, why does the `test_harminv` (in the same test file) do this?